### PR TITLE
:zap: Add Summary for Video_Representation_LLM

### PR DESCRIPTION
**Pull request by manyana72**

### CODE REVIEW

**Comment by mr-sarthakgupta** [comment]
Readme not updated. Please update readme

**Comment by mr-sarthakgupta** [comment]
@manyana72, please squash all the commits into one as instructed in the contributing guidelines.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ This repo houses summaries for various excitng works in the field of **Deep Lear
 - #### Photorealistic Text-to-Image Diffusion Models with Deep Language Understanding [[Paper](https://arxiv.org/abs/2205.11487)][[Review](./summaries/imagen.md)]
 
   - Chitwan Saharia, William Chan, Saurabh Saxena, Lala Li, Jay Whang, Emily Denton, Seyed Kamyar Seyed Ghasemipour, Burcu Karagol Ayan, S. Sara Mahdavi, Rapha Gontijo Lopes, Tim Salimans, Jonathan Ho, David J Fleet, Mohammad Norouzi, **NIPS 2022**
+  
+- #### Learning Video Representations from Large Language Models [[Paper](https://arxiv.org/pdf/2212.04501.pdf)][[Review](./summaries/Video_Representation_LLM.md)]
+
+  - Yue Zhao, Ishan Misra, Philipp Krähenbüh, Rohit Girdhar, Facebook AI Research- Meta AI, University of Texas, Austin, **Science Journal 2022**
```
Comment (praeclarumjj3) [inline README.md:35]: This one's published in Science Journal?

Comment (manyana72) [reply README.md]: Corrected that.

**Review by mr-sarthakgupta — APPROVED** [review]

**Pull request merged**